docs/language.rst

```diff
@@ -136,6 +136,79 @@ This also has a slightly more baroque syntax, which does exactly the same.
 
     import * as defs from "defines.sol";
 
+Builtin Imports
+_______________
+
+Some builtin functionality only available after importing. The following structs
```

Comment:
```suggestion
Some builtin functionality is only available after importing. The following structs
```

Reply:
Thanks for spotting that. Good reviews, I appreciate it!